save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -60,7 +57,6 @@ class VulnerabilityController(
         @RequestBody filter: VulnerabilityFilter,
         authentication: Authentication?,
     ): Mono<VulnerabilityMetadataDtoList> = blockingToMono {
-        checkPermission(filter, authentication)
```

Comment:
why do we remove permission checking?

Reply:
Because vulnerabilities should be displayed for all users (even without login). Additionally, it is checked on the frontend (depending on the role of the current user) whether only `APPROVED` and `AUTO_APPROVED` vulnerabilities or all should be displayed.

Reply:
as I get this checking, all users can get only approved vulnerabilities.
even we have checking on frontend, backend should be protected too since it's API and it can be accessed not only by FE.

Reply:
Done